NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Dtb delete post - Ticket 7

# Description

> As an author I would like the ability to remove a post I have written so that I can prevent others from reading it when I decide it is no longer something I wish people to read.
> 
> **Given** an author is viewing a Post that they have written
> **When** they select the delete option
> **Then** they should be presented to confirm the deletion
> 
> **Given** the author wishes to confirm the delete
> **When** they the select the option to confirm
> **Then** the Post should be removed from the system
> **And** the author should be directed back to the Post list
> 
> **Given** the author decides not to confirm the delete
> **When** they the select the option to reject confirmation
> **Then** the Post should NOT be removed from the system
> **And** the author should be directed back to the Post details


## Type of change
Please delete options that are not relevant.

# Testing Instructions
- login to the tabloid application
- add seed data to delete using a SQL query (my queries are saved in the project folder under SQLquery1.sql - there is an update method to update the isApproved column based on entry Id and an Insert method to add seed data)
- Select the Post tab in the Tabloid Application. Ensure that only posts posted by the logged in user display an option to Delete
- Select the MyPosts tab. Make sure that the Delete button appears for each entry
- Click on the Delete button. You should be taken to a confirmation page.
- Click Cancel. You should be taken back to /posts
- Click delete again. Then click Confirm. you should be taken back to /posts. Verify that the post was deleted from the list.

# Checklist:
